src/components/Property/Types/VectorProperty/ViewOptions/ColorView.tsx

```diff
@@ -1,9 +1,28 @@
-import { Flex, RGBA } from '@mantine/core';
+import { useState } from 'react';
+import { RGBA } from '@mantine/core';
+import { isNumber } from 'lodash';
 
 import { ColorPicker } from '@/components/ColorPicker/ColorPicker';
+import { StringInput } from '@/components/Input/StringInput';
+import { WarningIcon } from '@/components/WarningIcon/WarningIcon';
 import { AdditionalDataVectorMatrix } from '@/types/Property/propertyTypes';
+import { openspaceColorToRgba, toOpenspaceColor } from '@/util/colorHelper';
 
-import { VectorDefaultView } from './VectorDefaultView';
+function validateInput(
+  value: number[],
+  additionalData: AdditionalDataVectorMatrix,
+  isInt: boolean
+) {
+  if ((value.length !== 3 && value.length !== 4) || isInt) {
```

Comment:
```suggestion
  if (value.length !== 3 || value.length !== 4 || isInt) {
```
?

Reply:
Will always evaluate to true so not correct in this case

Reply:
Correct!  sorry for the noise :)